### PR TITLE
Add Chinese renering test

### PR DESCRIPTION
**Pull request by amyleew**
Checking tiles with different densities to see how the glyph atlas handles CJK font rendering.

/cc @xrwang

### CODE REVIEW

**Comment by amyleew** [comment]
@lucaswoj - These CJK character density test are passing. Can you take a look at them and give me a 👍  before merging them into master?

File: README.md [hunk 1]
```diff
@@ -3,15 +3,12 @@ Verify correctness and consistency of [mapbox-gl-js](https://github.com/mapbox/m
...
-and specifies values such as the map size, coordinates, and style classes. The expected output for a given test case
-is in `{id}/expected.png`, e.g. [`tests/background-color/constant/expected.png`](https://github.com/mapbox/mapbox-gl-test-suite/blob/master/tests/background-color/constant/expected.png).
+Within a leaf directory is a `style.json` file, which contains the minimal style needed for the given test case. The style can specify the map size, center, bearing, and pitch, and additional test metadata. The expected output for a given test case is in `expected.png`, e.g. [`tests/background-color/constant/expected.png`](https://github.com/mapbox/mapbox-gl-test-suite/blob/master/tests/background-color/constant/expected.png).
 
-Supporting files -- glyphs, sprites, and tiles -- live in their own respective subdirectories. The test
+Supporting files -- glyphs, sprites, and tiles -- live in their own respective subdirectories at the top level. The test
 harness sets up the environment such that requests for these resources are directed to the correct location.
 
 The contents of vector tile fixtures can be read using the [`vt2geojson`](https://github.com/mapbox/vt2geojson) tool
```
Comment (lucaswoj) [inline README.md]: Thank you for updating this! 😄

File: data/bbox2.geojson [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{ type: 'Feature',
+  properties: {},
+  geometry: 
+   { type: 'Point',
+     coordinates: [ -90.63720703125, 39.28329180414049 ] } }
```
Comment (lucaswoj) [inline data/bbox2.geojson]: These files ☝️ should have the `.json` extension, be valid JSON, and use the standard JSON formatting for this repo.

If you change the extension and then run `npm run jsonlint`, everything should be 👌 .

File: data/cjkPts.geojson [hunk 3]
```diff
@@ -0,0 +1 @@
..."properties":{"name":"躁邱凰溢椎"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.80376452461223,43.41299359444341]},"properties":{"name":"砸趟帘帆栖"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.73771479115749,43.38964563062126]},"properties":{"name":"窜丸斩堤塌"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.77914081236031,43.36200438697811]},"properties":{"name":"贩厢掀喀乖"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.6885010122187,43.3536150480067]},"properties":{"name":"谜捏阎滨虏"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.6809883295955,43.4279302964151]},"properties":{"name":"匙芦苹卸沼"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.66638088219406,43.356600228775875]},"properties":{"name":"钥株祷剖熙"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.81508020435649,43.44503598067335]},"properties":{"name":"哗劈怯棠胳"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.80562566482058,43.441746093418764]},"properties":{"name":"桩瑰娱娶沫"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-70.69680728921867,43.37119072206014]},"properties":{"name":"嗓蹲焚淘嫩"}}]}
```
Comment (lucaswoj) [inline data/cjkPts.geojson]: Why do we need so many new GeoJSON fixtures for this test? Can we refactor to use a single fixture or rename the fixtures to clearly explain the differences between them?

`bbox.geojson`, `bbox1.geojson`, `bbox2.geojson`, `bbox3.geojson`, `bbox4.geojson`, `data/cjk.geojson`, and `data/cjkPts.geojson`

File: render-tests/cjk-fonts/literal/style.json [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        "icon-image": "dog-park-12",
+        "text-field": "{name}",
+        "text-font": [
+          "NotoCJK"
+        ],
+        "symbol-placement": "point"
+      }
+    }
+  ]
+}
```
Comment (lucaswoj) [inline render-tests/cjk-fonts/literal/style.json]: Is the expected output correct? Based on the geojson fixtures I would expect to see CJK glyphs not, for example, roman numerals.

Comment (lucaswoj) [inline render-tests/cjk-fonts/literal/style.json]: Why did you choose to create a new directory (`cjk-fonts`) rather than `text-field` or `text-font`?

**Comment by lucaswoj** [comment]
Just force-pushed to get rid of some unrelated commits that found their way into this branchj.

**Comment by lucaswoj** [comment]
Is there any way to pare down the files in `glyphs/NotoCJK`? The directory adds 15mb to the repo.

File: render-tests/cjk-fonts/test4/style.json [hunk 5]
```diff
@@ -0,0 +1,43 @@
+{
```
Comment (lucaswoj) [inline render-tests/cjk-fonts/test4/style.json]: ☝️  Is a blank map really what's "expected" from `test4`?

**Comment by amyleew** [comment]
@lucaswoj - Per chat with @xrwang, she's looking into slimming down the `glyphs/NotoCJK`. We need all the fonts but may have some wiggle room there.

I'll hit the other clarifications for the tests.

File: render-tests/cjk-fonts/literal/style.json (hunk 4 above)
Comment (amyleew) [inline render-tests/cjk-fonts/literal/style.json]: Didn't think about that as an option. I did want to isolate the tests to one folder, but I don't see a problem with rolling them into either `text-field` or `text-font`

**Comment by amyleew** [comment]
Per chat w/ @xrwang and @lucaswoj - I'm gonna hit the `.pbf` updates !

**Comment by xrwang** [comment]
@lucaswoj pared down the pbfs as much as possible, to only cover CJK ranges in Noto.

**Comment by lucaswoj** [comment]
This is ready to ship in conjunction with https://github.com/mapbox/mapbox-gl-native/issues/5873 https://github.com/mapbox/mapbox-gl-js/pull/2930 and https://github.com/mapbox/mapbox-gl-shaders/pull/22!

**Comment by lucaswoj** [comment]
Pushing a squash & rebase. Previous HEAD was https://github.com/mapbox/mapbox-gl-test-suite/pull/131/commits/f7428423fcf28ca2699ba98cb68b5bf5262334ab

**Pull request merged**